service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,399 @@
+                <set field="orderMap" from="[orderId:order_detail.orderId,orderName:order_detail.orderName,currencyUom:order_detail.currencyUom.uomId,
+                                             salesChannelEnumId:order_detail.salesChannelEnumId,statusId:order_detail.statusId,placedDate:order_detail.placedDate,
+                                             grandTotal:order_detail.grandTotal,customer_details:customerMap,order_parts:partList]"/>
+            </iterate>
+            <script>order.add(orderMap)</script>
+        </actions>
+    </service>
+    <!-- get all order details here  -->
+    <service verb="get" noun="ReturnAllOrders">
+        <out-parameters>
+            <parameter name="orders" type="List">
+                <parameter name="orderMap" type="Map">
+                    <parameter name="orderId"/>
+                    <parameter name="orderName"/>
+                    <parameter name="currencyUom"/>
+                    <parameter name="salesChannelEnumId"/>
+                    <parameter name="statusId"/>
+                    <parameter name="placedDate"/>
+                    <parameter name="grandTotal"/>
+                    <parameter name="customer_details" type="List">
+                        <parameter name="customerMap" type="Map">
+                            <parameter name="customerPartyId"/>
+                            <parameter name="firstName"/>
+                            <parameter name="middleName"/>
+                            <parameter name="lastName"/>
+                        </parameter>
+                    </parameter>
+                    <parameter name="order_parts" type="List">
+                        <parameter name="partMap" type="Map">
+                            <parameter name="orderPartSeqId"/>
+                            <parameter name="partName"/>
+                            <parameter name="facilityId"/>
+                            <parameter name="shipmentMethodEnumId"/>
+                            <parameter name="partStatusId"/>
+                            <parameter name="partTotal"/>
+                            <parameter name="item_details" type="List">
+                                <parameter name="itemMap" type="Map">
+                                    <parameter name="orderItemSeqId"/>
+                                    <parameter name="productId"/>
+                                    <parameter name="itemDescription"/>
+                                    <parameter name="quantity"/>
+                                    <parameter name="unitAmount"/>
+                                </parameter>
+                            </parameter>
+                        </parameter>
+                    </parameter>
+                </parameter>
+            </parameter>
+        </out-parameters>
+        <actions>
+            <!-- Fetch data from orderHeader -->
+            <entity-find entity-name="mantle.order.OrderHeader" list="orderList">
+
```

Comment:
Please remove extra spaces wherever not needed.

Reply:
Okay, Ma'am.